nac/workflows/workflow_ipr.py

```diff
@@ -0,0 +1,82 @@
+__all__ = ['workflow_ipr']
+
+# Module for single_point workflow
+from nac.workflows import workflow_single_points
+
+# Modules for IPR caluclation
+import h5py
+from nac.common import (
+    number_spherical_functions_per_atom,
+    retrieve_hdf5_data)
+from nac.integrals.multipole_matrices import compute_matrix_multipole
+import numpy as np
+from qmflows.parsers.xyzParser import readXYZ
+from scipy.linalg import sqrtm
+from nac.workflows.initialization import initialize
+
+import logging
+
+# Starting logger
+logger = logging.getLogger(__name__)
+
+
+def workflow_ipr(config: dict) -> list:
+
+    # Dictionary containt the general information
+    config.update(initialize(config))
+
+    # Logger info
+    logger.info("Starting single point calculation.")
+
+    # Call the single point workflow to calculate the eigenvalues and
+    # coefficients
+    hdf5_path = workflow_single_points(config)
+
+    # Logger info
+    logger.info("Starting IPR calculation.")
+
+    # Get eigenvalues and coefficients from hdf5 file
+    c_AO = retrieve_hdf5_data(config["path_hdf5"], hdf5_path[0][0][1])
+    Energies = retrieve_hdf5_data(config["path_hdf5"], hdf5_path[0][0][0])
+    Energies = Energies * 27.211  # To get them from Hartree to eV
```

Comment:
Use the constant define in [scipy](https://docs.scipy.org/doc/scipy/reference/constants.html) for instance: https://github.com/SCM-NV/qmflows-namd/blob/master/nac/common.py#L48